ui/src/app/applications/components/applications-list/applications-list.tsx

```diff
@@ -247,7 +247,10 @@ export const ApplicationsList = (props: RouteComponentProps<{}>) => {
                 return syncApp && Observable.fromPromise(services.applications.get(syncApp)) || Observable.from([null]);
              }) }>
             {(app) => (
-                <ApplicationSyncPanel key='syncPanel' application={app} selectedResource={'all'} hide={() => ctx.navigation.goto('.', { syncApp: null })} />
+                <ApplicationSyncPanel key='syncPanel' application={app} selectedResource={'all'} hide={() => ctx.navigation.goto('.', { syncApp: null })}
+                update={(ignoreWarnings: boolean) => {
```

Comment:
does this mean we cannot use this form the applications list?

Reply:
I may be wrong, but I only found sync warnings to only be displayed in `application-details.tsx`.